plugin/trino-iceberg/pom.xml

```diff
@@ -374,6 +380,36 @@
             <scope>runtime</scope>
         </dependency>
 
+        <dependency>
```

Comment:
These shouldn't be needed, as they are dependencies of `trino-filesystem-s3`. I verified that they get pulled in as runtime dependencies (as expected).